utils/components.py

```diff
@@ -1,5 +1,4 @@
 # type: ignore
-# pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error,no-member,missing-function-docstring
```

Comment:
Why is this gone?

Reply:
It seems it was not needed anymore... Or at least my local machine does not report any errors anymore. 🤷